src/main/java/at/hannibal2/skyhanni/features/inventory/auctionhouse/AuctionHouseOpenPriceWebsite.kt

```diff
@@ -0,0 +1,89 @@
+package at.hannibal2.skyhanni.features.inventory.auctionhouse
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.GuiContainerEvent
+import at.hannibal2.skyhanni.events.InventoryCloseEvent
+import at.hannibal2.skyhanni.events.InventoryFullyOpenedEvent
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.NEUInternalName.Companion.asInternalName
+import at.hannibal2.skyhanni.utils.NEUItems.getItemStack
+import at.hannibal2.skyhanni.utils.OSUtils
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import io.github.moulberry.notenoughupdates.events.ReplaceItemEvent
+import io.github.moulberry.notenoughupdates.util.Utils
+import net.minecraft.entity.player.InventoryPlayer
+import net.minecraft.item.ItemStack
+import net.minecraftforge.fml.common.eventhandler.EventPriority
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.seconds
+
+class AuctionHouseOpenPriceWebsite {
+
+    private val config get() = SkyHanniMod.feature.inventory.auctions
+    private var lastClick = SimpleTimeMark.farPast()
+
+    private val patternGroup = RepoPattern.group("inventory.auctionhouse")
+
+    /**
+     * REGEX-TEST: Auctions: "hyperion"
+     */
+    private val ahSearchPattern by patternGroup.pattern(
+        "title.search",
+        "Auctions: \"(?<searchTerm>.*)\""
```

Comment:
Does not match long patterns

![image](https://github.com/hannibal002/SkyHanni/assets/94038482/7283f200-ef08-41ed-982d-fe61886825f3)